packages/block-library/src/footnotes/index.php

```diff
@@ -89,14 +104,9 @@ function register_block_core_footnotes() {
 			);
 		}
 	}
-	register_block_type_from_metadata(
-		__DIR__ . '/footnotes',
-		array(
-			'render_callback' => 'render_block_core_footnotes',
-		)
-	);
 }
-add_action( 'init', 'register_block_core_footnotes' );
+// Use a priority of 20 to be higher than the default of 10 the priority with which most post types are registered.
```

Comment:
```suggestion
// Most post types are registered at priority 10, so use priority 20 here in
// order to catch them.
```

Reply:
This will need to use the coding standards format for multiline comments.